io/net/infiniband/ip_over_ib.py

```diff
@@ -55,6 +55,22 @@ def setUp(self):
         self.NETSERVER_RUN = self.params.get("NETSERVER_RUN", default="0")
         self.iper = os.path.join(self.srcdir, 'iperf')
         self.netperf = os.path.join(self.srcdir, 'netperf')
+
+        detected_distro = distro.detect()
+        if detected_distro.name == "Ubuntu":
+            cmd = "service ufw stop"
+        elif detected_distro.name in ['redhat', 'fedora']:
+            cmd = "systemctl stop firewalld"
+        elif detected_distro.name == "Suse":
+            cmd = "rcSuSEfirewall2 stop"
+        elif detected_distro.name == "centos":
+            cmd = "service iptables stop"
+        else:
+            self.skip("Distro not supported")
+        if process.system("%s && ssh %s %s" % (cmd, self.PEER_IP, cmd),
+                          ignore_status=True, shell=True) != 0:
+            self.skip("Unable to disable firewall")
+
```

Comment:
@PraveenPenguin oh okay thanks for correcting it.   I thought it would be easier to read the code so had added it. will make sure this doesn't repeat